### PR TITLE
Footer alpha 6

### PR DESCRIPTION
**Pull request by alanmoo**
@mmmavis Can you rebase your branch off master? That should make this auto-mergable

### CODE REVIEW

**Comment by mmmavis** [comment]
@alanmoo done rebasing. There seems to be conflicts still in this PR though. 👀

**Comment by alanmoo** [comment]
I'm not sure why that was happening, but it's been resolved now.

File: src/index.pug [hunk 1]
```diff
@@ -226,28 +226,28 @@ html
 
     footer
       .container
-        ul.footer-links 
-          li 
+        ul.list-unstyled.footer-links.row.justify-content-center
+          li.col-auto
```
Comment (mmmavis) [inline src/index.pug]: Do we need to assign `<li>` `.col-auto` class here? Layout seems fine without those extra rules in `col-auto`. Or I guess it's a good practice to assign immediate children of a `.row` element with `col-*` class?

Comment (mmmavis) [inline src/index.pug]: @gvn @alanmoo ^ ?

Comment (alanmoo) [inline src/index.pug]: I'd rather be a bit explicit about it, and yeah, it's good practice as a reminder that this is handled by the grid.

Comment (mmmavis) [inline src/index.pug]: cool just wanted to double check 👍

**Pull request merged**